apps/src/code-studio/pd/teachercon1819_registration/Teachercon1819Registration.jsx

```diff
@@ -0,0 +1,107 @@
+import {PropTypes} from 'react';
+import FormController from '../form_components/FormController';
+
+import Welcome from './Welcome';
+import Joining from './Joining';
+import TravelPlans from './TravelPlans';
+import Releases from './Releases';
+import Confirmation from './Confirmation';
+
+import { TeacherSeatAcceptanceOptions } from '@cdo/apps/generated/pd/teachercon1819RegistrationConstants';
+
+export default class Teachercon1819Registration extends FormController {
+  static propTypes = {
+    ...FormController.propTypes,
+    applicationId: PropTypes.number.isRequired,
+    applicationType: PropTypes.string.isRequired,
+    course: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    date: PropTypes.string.isRequired,
+    email: PropTypes.string.isRequired,
+    firstName: PropTypes.string,
+    lastName: PropTypes.string,
+    phone: PropTypes.string,
+  };
+
+  constructor(props) {
+    super(props);
+
+    // initialize some fields with reasonable defaults
+    this.state.data = {
+      ...this.state.data,
+      email: this.props.email,
+      preferredFirstName: this.props.firstName,
+      lastName: this.props.lastName,
+      phone: this.props.phone
+    };
+  }
+
+  //static sessionStorageKey = "Teachercon1819Registration";
```

Comment:
intentional?

Reply:
oh, thanks! I disabled it for testing and forgot to mark with TODO